generator-cas-app/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2021 Naomi Aro <naomi.aro@gov.bc.ca>
```

Comment:
maybe copy the LICENSE file that is at the root of this project? It should probably be copyrighted with "Government of British Columbia <ggircs@gov.bc.ca>" (the license file at the root should be updated too as it has a placeholder year and owner, it is probably the same in all of our repos :upside_down_face: )

Reply:
This shorter one is maybe the license that should be applied at both levels? http://www.apache.org/licenses/LICENSE-2.0#apply It was auto generated by the yeoman scaffold like this, but I should swap my name out for sure.